compiler/test-resources/repl/i2997

```diff
@@ -0,0 +1,8 @@
+scala> trait A
```

Comment:
Wrong testcase?

Reply:
Wrong issue. I meant #2977 😃. Will update the PR